src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,76 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String COMMA = ",";
+    private static final String RESULT = "result";
+    private static final int LENGTH_OF_WORD_SUPPLY = 7;
+    private static final int LENGTH_OF_WORD_BUY = 4;
 
+    public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
As described in common mistakes, you should divide your logic into separate methods. I'd recommend creating separate methods for reading, report generating, and writing